src/plugins/intel_cpu/src/utils/general_utils.h

```diff
@@ -11,6 +11,18 @@
 
 #include "openvino/core/type/element_type.hpp"
 
+
+#if defined(__GNUC__) || defined(__clang__)
+#define ov_unlikely(x)     (__builtin_expect(!!(x), false))
+#define ov_likely(x)       (__builtin_expect(!!(x), true))
+#elif(defined(__cplusplus) && (__cplusplus >= 202002L)) || (defined(_MSVC_LANG) && _MSVC_LANG >= 202002L)
+#define ov_unlikely(x)     (x) [[unlikely]]
+#define ov_likely(x)       (x) [[likely]]
+#else
+#define ov_unlikely(x)     (x)
+#define ov_likely(x)       (x)
+#endif
+
```

Comment:
It seems that we don't really need these optimizations as their usage doesn't bring a lot of perf benefits and on the other hand they may be misused due to wrong predictions of the probability branch taking.